browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -342,6 +342,52 @@ function getAbsolutePathsOfAttachmentsInContent (markdownContent, storagePath) {
   return result
 }
 
+/**
+ * @description Copies the attachments to the storage folder and returns the mardown content it should be replaced with
+ * @param {String} markDownContent content in which the attachment paths should be found
+ * @param {String} filepath The path of the file with attachments to import
+ * @param {String} storageKey Storage key of the destination storage
+ * @param {String} noteKey Key of the current note. Will be used as subfolder in :storage
+ */
+function importAttachments (markDownContent, filepath, storageKey, noteKey) {
+  return new Promise((resolve, reject) => {
+    const nameRegex = /(!\[.*?]\()(.+?\..+?)(\))/g
+    let attachName = nameRegex.exec(markDownContent)
+    const promiseArray = []
+    const attachPath = []
+    const groupIndex = 2
+
+    while (attachName) {
+      attachPath.push(attachName[groupIndex])
```

Comment:
Can you change the name `attachName[groupIndex]` into something like `attachmentPath`? Also if you do that, you might want to change the `attachPath` variable into `attachmentPaths` (with the `s`). Also with the current code, it only supports absolute attachment. Can you also support relative path? You can check the relative or absolute path using [`path.isAbsolute`](https://nodejs.org/api/path.html#path_path_isabsolute_path) in the `path` module.